src/main/charts/bitbucket/templates/statefulset.yaml

```diff
@@ -178,6 +178,8 @@ spec:
             {{- include "bitbucket.databaseEnvVars" . | nindent 12 }}
             {{- include "bitbucket.sysadminEnvVars" . | nindent 12 }}
             {{- include "bitbucket.elasticSearchEnvVars" . | nindent 12 }}
+            - name: PLUGIN_SSH_PORT
```

Comment:
I see `plugin.ssh.port` is mentioned here: I see `plugin.ssh.port` is a property that can be supplied: https://community.atlassian.com/t5/Bitbucket-questions/Custom-ssh-port-for-bitbucket-smart-mirror/qaq-p/2131340.  Was just wondering if there was any official documentation regarding its use?